var/spack/repos/builtin/packages/py-elephant/package.py

```diff
@@ -28,40 +31,53 @@ class PyElephant(PythonPackage):
     version("0.3.0", sha256="747251ccfb5820bdead6391411b5faf205b4ddf3ababaefe865f50b16540cfef")
 
     variant("docs", default=False, description="Install documentation dependencies")
-    variant("pandas", default=False, description="Build with pandas", when="@0.3.0:0.4.1")
```

Comment:
@alalazo another example of a variant being removed from a package. Could really use a formal variant deprecation scheme: #38829